src/app/Scenes/MyCollection/Screens/ArtworkList/MyCollectionArtworkGridItem.tsx

```diff
@@ -32,6 +38,47 @@ const MyCollectionArtworkGridItem: React.FC<MyCollectionArtworkGridItemProps> =
   const sectionCount = isPad() ? (isPadHorizontal ? 4 : 3) : 2
   const imageWidth = (screen.width - DEFAULT_SECTION_MARGIN * (sectionCount + 1)) / sectionCount
 
+  const isPOneArtist =
+    !!artwork.artists?.find((artst) => Boolean(artst?.targetSupply?.isTargetSupply)) ??
+    !!artwork.artist?.targetSupply?.isTargetSupply ??
+    false
+
+  const ARShowDemandIndexHints = useFeatureFlag("ARShowDemandIndexHints")
+
+  const fetchDemandRank = async (): Promise<number | null> => {
+    if (artist?.internalID && medium) {
+      try {
+        const res = await fetchQuery<MyCollectionArtworkGridItemQuery>(
+          defaultEnvironment,
+          graphql`
+            query MyCollectionArtworkGridItemQuery($internalID: ID!, $medium: String!) {
+              marketPriceInsights(artistId: $internalID, medium: $medium) {
+                demandRank
+              }
+            }
+          `,
+          { internalID: artist.internalID, medium }
+        ).toPromise()
+
+        if (res?.marketPriceInsights?.demandRank) {
+          setTrending(Number(res.marketPriceInsights.demandRank * 10) >= 9)
```

Comment:
```suggestion
         const calculatedTrending = Number(res.marketPriceInsights.demandRank * 10) >= 9
          setTrending(Number(rcalculatedTrending)
```
I think this way its easier to understand 😬
Up to you though.